### PR TITLE
Support for Kubernetes v1.20

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area open-source usability
/kind enhancement
/priority normal
/platform aws
/exp intermediate
/topology garden seed shoot

**What this PR does / why we need it**:
This PR adds support for 1.20 to the extension.

**Which issue(s) this PR fixes**:
Part of gardener/gardener#3260

**Special notes for your reviewer**:
* :white_check_mark: Depends on gardener/gardener#3296
* I have successfully validated the functionality as follows:
  * :white_check_mark: Create new clusters with versions < 1.20
  * :white_check_mark: Create new clusters with version  = 1.20
  * :white_check_mark: Upgrade old clusters from version 1.19 to version 1.20
  * :white_check_mark: Delete clusters with versions < 1.20
  * :white_check_mark: Delete clusters with version  = 1.20

**Release note**:

```feature user
The AWS extension does now support shoot clusters with Kubernetes version 1.20. You should consider the [Kubernetes release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md) before upgrading to 1.20. 
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This extension controller supports the following Kubernetes versions:
 
 | Version         | Support     | Conformance test results |
 | --------------- | ----------- | ------------------------ |
+| Kubernetes 1.20 | 1.20.0+     | N/A |
```
Comment (ialidzhikov) [inline README.md:26]: Can you update the link in the PR release note:

```diff
- https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.20.md
+ https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md
```

Comment (rfranzke) [reply README.md:26]: Thanks for catching this! 💕 Done

**Comment by rfranzke** [comment]
/ready as https://github.com/gardener/gardener/pull/3296 got approved and merged now

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**